martin.yaml

```diff
@@ -7,15 +7,35 @@ max_feature_count: 500
 postgres:
     connection_string: ${DATABASE_URL}
     functions:
-        treatment_plan_prescriptions:
+        # layer with a series of stands, given a treatment_plan_id
+        # parameters:
+        # - treatment_plan_id: int
+        # - project_area_id: Optional[int]
+        dynamic_stands:
             schema: public
-            function: martin_get_treatment_plan_prescriptions
+            function: martin_dynamic_stands
             minzoom: 0
             maxzoom: 20
             bounds: [-180.0, -90.0, 180.0, 90.0]
-        project_area_outline:
+        # layer with the project area outline. This unions all stands
+        # that compose the project are and creates a single multi polygon.
+        # this layer changes the shape of the project area to conform to
+        # the underlying stands.
+        # parameters:
+        # - project_area_id: int
+        dynamic_project_area:
             schema: public
-            function: martin_project_area_outline
+            function: martin_dynamic_project_area
+            minzoom: 0
+            maxzoom: 20
+            bounds: [-180.0, -90.0, 180.0, 90.0]
+        # layer with project areas. This layer shows original project
+        # areas, as imported or generated by any algorithm.
+        # parameters:
+        # - scenario_id: int
+        dynamic_project_areas:
```

Comment:
im a bit confused on this, is this one called the same name as the one above? I'll suggest something like scenario_project_areas